libraries/cloudharness-common/cloudharness/events/client.py

```diff
@@ -7,21 +7,37 @@
 
 from time import sleep
 from json import dumps, loads
+from keycloak.exceptions import KeycloakGetError
 from kafka import KafkaProducer, KafkaConsumer
 from kafka.admin import KafkaAdminClient, NewTopic
 from kafka.errors import TopicAlreadyExistsError, UnknownTopicOrPartitionError, KafkaTimeoutError
 
 from cloudharness import log
+from cloudharness.auth.keycloak import AuthClient
 from cloudharness.errors import *
 from cloudharness.utils import env
+from cloudharness.utils.config import CloudharnessConfig as config
 
 logging.getLogger('kafka').setLevel(logging.ERROR)
 
+
+AUTH_CLIENT = None
+CURRENT_APP_NAME = config.get_current_app_name()
+
+def get_authclient():
+    global AUTH_CLIENT
+    if not AUTH_CLIENT:
+        AUTH_CLIENT = AuthClient()
+    return AUTH_CLIENT
+
+
 class EventClient:
-    def __init__(self, topic_id):
-        self.client_id = env.get_cloudharness_events_client_id()
+    def __init__(self, topic_id, create_topic=False):
         self.topic_id = topic_id
+        self.client_id = env.get_cloudharness_events_client_id()
```

Comment:
I think that this and below usage of the env api should be deprecated. Those values are coming from te values.yaml config rather than environment variables.

Reply:
I already refactored that a while ago. It now reads the values.yaml and returns the value

Reply:
I mean that the api itself comes from an old approach, when the values.yaml as a configmap wasn't a thing. Now we can get an application configuration and we can ask values to the application directly. This is just an alternative way to do the same thing.

Reply:
done